examples/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# create examples
+if(TS_CPP_AVAILABLE)
```

Comment:
I think there is no need to export `TS_CPP_AVAILABLE` and check for it. Instead, one can simply check for `tinysplinecpp_static`:

```bash
if (TARGET tinysplinecpp_static)
...
```